### ISSUE TITLE
Aanvullende databronnen

### ISSUE THREAD

**Issue by sjerpbouwtsites** [sjerpbouwtsites/faillissementen-scraper#9]
BAG + kvK + koppeltabel
zie github NL extract

**Comment by sjerpbouwtsites** [comment]
gebruiker meld: https://www.fundainbusiness.nl/alle-bedrijfsaanbod/amsterdam/per-direct/

**Comment by jellespijker** [comment]
Kreeg je site doorgestuurd van een vriend van mij omdat dit een aantal gelijkenissen heeft met een projectje waar ik al een tijdje me in mijn hoofd speel.

Zelf was ik van plan Street map en satelliet beelden met computer vision algorithms te analyseren door middel van gebruiken met een Neural Network of ander machine learning algorithm. Dit in combinatie met faillissement data, type gebied (kantoor, woon, industrie) inkomensindex, bevolkingsmigratie enz.

Ik ben zelf een ouwe kraker (alweer 15 jaar geleden) nu alleen nog urbexen. Daar was ik ook voor bezig, veelal in België.

Ik heb je code en je gebruikte bronnen nog niet bekeken dus wellicht heb je hier al aangebracht.
Tegenwoordig is veel data van de EU landen opensource. Zo kan je voor Antwerpen bijvoorbeeld een kaart voor de leegstand vinden onder.

 https://portaal-stadantwerpen.opendata.arcgis.com/datasets/891bdc6ad9664a0f9167460cebfaf3ed_201 

Alle opendata van de Nederlandse overheid kan je hier vinden:
https://data.overheid.nl/

Opendata België:
https://data.gov.be/nl

**Comment by huib23** [comment]
Dit zijn databronnen die ik gebruik sommige kunnen in de kaart interface worden geimplemteerd andere zal het nice zijn om miss een soort checklist interface voor te hebben, waarbij sommige pas te gebruiken zijn als je weet wie de eigenaar van het pand is (als dus de koopakte is aangeschaft). Sommige zal je op dat moment opnieuw willen gebruiken (en zalden dan dus miss 2 keer op zon checklist moeten terug komen)

ruimtelijkeplannen.nl
Bestemings plannen zijn hier te vinden hierin wordt gespecificeerd wat vorn funtie een gebouw mag hebben (horeca/wonen/enz.)

https://odnzkg.nazca4u.nl/rapportage/viewerLookUp/Geolocator.aspx
Omgevingsdiens noordzeekanaal gebied.
Moeilijk te lezen raporten zijn hier te generen over de vervuilingen van percelen (en de zogenoemde BUS meldingen hier over) (Alleen in n&z-holland geloof ik)

bagviewer.nl 
Kaart met percellen, en bijhoorende adressen.

Webshop kadaster
Eigedoms akte verteld wie de eigenaar van pand/perceel is. (Kosten 2,90 EUR)

https://www.crk.amsterdam.nl/  (Aleen amsterdam)
Vertelt over mulumentale panden (en panden met somigen andere labels) informatie over vergunning aanvragen. Het vertelt niks over bestuur rechelijke beslisselingen alleen over of de commesie de molumentale waarde geresprecteerd vind blijven. het vertelt wat de vergunning aanvraag inhoudt tot op regelijk hoog detail en ook waarom deze wordt geacpteerd of geweigerd.

oozo.nl,
Vertelt de meta informatie over vergunning aanvragen (vertelt: aanvraag onderwerp, datum aanvraag ontvangen, aanvraag uitlag uitgesteld, gewijgerd/aangenomen) (partucileren site baseert op veel lokale vergunning data)

https://www.amsterdam.nl/nieuws/kennisgevingen-bekendmakingen/ (lokale vergunning data van amsterdam)
Vertelt de meta informatie over vergunning aanvragen (vertelt: aanvraag onderwerp, datum aanvraag ontvangen, aanvraag uitlag uitgesteld, gewijgerd/aangenomen (gaat alleen tot 6 maanden terug) (hier is oozo dus bv op gebaseerd als ik me niet vergis)

drimble.nl
Vertelt zelfde als bovengenoemde (minder compleet) 
Daarnaast vertelt het iets wat je nergens anders zonder te betalen kan vinden en dat is de bestuursvoorzitter namen van een organisatie zoals geregisteerd in kvk.  (particulere site)

kvk.nl
Regestraties van bedrijven. adressen waar een bedrijf is ingeschren leidt soms naar bedrijven zelfde eigenaar

**Comment by sjerpbouwtsites** [comment]
@jellespijker 
> Neural Network of ander machine learning algorithm. 
Ja, dat klinkt wel aardig ambitieus. Klinkt als een megagroot project. Wat is de reden dat je er niet mee bent verder gegaan?
Deze app zoals hij nu is, is bewust simpel en ruw gebouwd. Ik heb iets te veel ambitieuze projecten gemaakt waar uiteindelijk geen vraag voor bleek 🤣 of ik had de vraag helemaal niet begrepen. Dus vandaar de v0, een testrun voor feedback en marktonderzoek.
 
> Ik ben zelf een ouwe kraker (alweer 15 jaar geleden) nu alleen nog urbexen. Daar was ik ook voor bezig, veelal in België.
Ja precies. Vind ik zelf ook wel leuk. Geinig.

> data van EU landen opensource. 
Ik heb daar aan gedacht, en zal kijken of het in de v1 een nuttige toevoeging kan zijn. 
De reden dat ik uitkwam op faillissementen met de kvk is... omdat dat mijn initieele idee was en die ging ik testen. Of het uberhaupt kon bijvoorbeeld. Er was geen principieele keuze in. Thx voor de tip!

**Comment by sjerpbouwtsites** [comment]
Zie verder
https://github.com/sjerpbouwtsites/faillissementen-scraper/issues/19

**Comment by jellespijker** [comment]
@sjerpbouwtsites het zou idd een groot project worden en Google had toen ook net zijn voorwaarden veranderd voor het dl van sateliet beelden.

Ik heb een kaart van een vriend welke aardig wat urbex locaties geïdentificeerd heeft. Dus proces zou zijn: binnen halen meerdere datasets, convert these to a GEOraster, waarbij iedere "pixel" bestaat uit een satelietfoto, en data uit de verschillende bronnen, (soort terrein industrie/woonwijk etc, inkomenslevel, bevolkingssamenstelling, aantal fallisementen, enz.). Iedere "pixel" krijgt de classificatie waarschijnlijkheid van urbex locatie; Gooi deze "pixels" door tensorflow oid, Waarbij onderstaande kaart als leer/validatie/test set kan dienen.

![image](https://user-images.githubusercontent.com/8535734/89507521-e7c57880-d7cc-11ea-865d-3973017153ef.png)
